packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.html

```diff
@@ -0,0 +1,18 @@
+<div class="sf-add-to-cart" data-test="sf-add-to-card">
+  <slot name="add-to-cart-btn">
+    <!--@slot Custom content that will replace default Add to cart button design.-->
+    <SfButton v-on="$listeners" class="sf-add-to-cart__button" :disabled="!canAddToCart">
+        Add to cart
+    </SfButton>
+  </slot>
+  <slot name="quantity-select-input" v-bind="{ qty }">
+    <SfInput 
+      :value="qty" 
+      v-on:change="$emit('input', qty)"
```

Comment:
```suggestion
    @input="(value)=>{$emit('input', value)}"
```
becouse SfInput use v-model :) and we shoud use :value and @input